src/App.js

```diff
@@ -24,9 +26,14 @@ function App() {
           <NavBar />
           <Content style={{ background: "white", padding: "0 5px" }}>
             <div className="site-layout-content">
+              <Route
+                exact
+                path="/story/:id"
```

Comment:
if the route for blogs is /blog then the route for this one should be /blog/:id its needs to be consistence

Reply:
It's same however below blogs have a text. However when i try to add another image to wordpress i can not get them because of that I created similar component and one of is row other one is column

Reply:
when I write blog is not working  I will change it as blogs.